aten/src/ATen/test/tensor_interop_test.cpp

```diff
@@ -0,0 +1,140 @@
+#include "gtest/gtest.h"
+
+#include "ATen/ATen.h"
+#include <caffe2/core/init.h>
+#include <caffe2/core/operator.h>
+
+TEST(TestTensorInterop, Caffe2ToPytorchSimple) {
+  caffe2::Tensor c2_tensor(caffe2::CPU);
```

Comment:
This is fine to test as long as we support it, but @jerryzh168 is the process of expunging all of these. So we might also want to test `caffe2::empty(dims, options)`, which is the modern way to allocate Caffe2 tensors and is slowly subsuming old uses (since `Output()` on operator now uses it.)